.github/PULL_REQUEST_TEMPLATE/manual_maintenance.md

```diff
@@ -80,6 +80,7 @@ Execute `sudo apt-get update && sudo apt-get upgrade` on
   - [ ] It's verified that real data has been used
   - [ ] It's verified that everything seems to be working fine by manually using the website
 - [ ] This template has been updated to reflect the latest state of tasks required and known issues with upgrades
+- [ ] At least one commit needs to be called 'manual-maintenance' to satisfy the CI maintenance check in future commits
```

Comment:
That won't help - the MR commit needs to have "manual maintenace" in the description

Reply:
Adapted the reminder.

Reply:
ok